Makefile

```
@@ -1,28 +1,32 @@
 build:
-	CGO_ENABLED=0 go build -o bin/istio-plugin -ldflags "-s -w" .
+	CGO_ENABLED=0 go build -o bin/load-balancer-webhook -ldflags "-s -w" .
```

should be switched back to istio-plugin

```suggestion
	CGO_ENABLED=0 go build -o bin/istio-plugin -ldflags "-s -w" .
```

---

Yep, copy and paste moment.